src/WebJobs.Script.WebHost/Diagnostics/DiagnosticEventTableStorageRepository.cs

```diff
@@ -184,7 +184,7 @@ public void WriteDiagnosticEvent(DateTime timestamp, string errorCode, LogLevel
             {
                 ErrorCode = errorCode,
                 HelpLink = helpLink,
-                LastTimeStamp = timestamp,
+                TimeStamp = timestamp,
```

Comment:
nit question: Should the constructor of `DiagnosticEvent` also set the ~Timestamp` property? I see we are already passing that piece of data?

Reply:
Good point, let me double check if need setters to be called explicitly.